tools/cli/commands/generate.js

```diff
@@ -440,6 +452,10 @@ async function generatePluginFromStarter( projDir, answers ) {
 	composerJson.extra ||= {};
 	composerJson.extra.changelogger ||= {};
 	composerJson.extra.changelogger.versioning = answers.versioningMethod;
+	// Add proposed WP.org slug if we've indicated this is a public plugin.
+	if ( answers.mirrorrepo ) {
+		composerJson.extra[ 'wp-plugin-slug' ] = normalizeSlug( answers.name );
```

Comment:
Should we also delete `beta-plugin-slug` in this case? There's not really a need for both, `beta-plugin-slug` is intended for plugins that can't have a `wp-plugin-slug` since they won't be public.

Reply:
Makes sense: d9c4ffb60d